python/llm/example/GPU/LangChain/vllm.py

```diff
@@ -0,0 +1,31 @@
+from ipex_llm.langchain.vllm.vllm import VLLM
```

Comment:
is multi-cards VLLM TP supported?
we may also need a example for  multi-cards VLLM TP

Reply:
Yes, it is supported. Will add multi-cards vLLM TP instructions